sbi/utils/torchutils.py

```diff
@@ -313,6 +313,11 @@ def __init__(
         # Device handling
         device = low.device.type if device is None else device
         device = process_device(device)
+        self.device = device
+        self.reinterpreted_batch_ndims = reinterpreted_batch_ndims
+
+        self.low = torch.as_tensor(low, dtype=torch.float32, device=device)
```

Comment:
Type hint suggests that the input is already a tensor.

If you would like to support also e.g. arrays then add them as type hint.